src/EditorFeatures/CSharpTest/ReplaceDefaultLiteral/ReplaceDefaultLiteralTests.cs

```diff
@@ -54,40 +54,26 @@ private async Task TestMissingWithLanguageVersionsAsync(string initialMarkup, Im
         [Fact]
         public async Task TestCSharp7_1_InCaseSwitchLabel_Int()
         {
-            await TestWithLanguageVersionsAsync(
+            await TestMissingWithLanguageVersionsAsync(
 @"class C
 {
     void M()
     {
         switch (1) { case [||]default: }
```

Comment:
Is the input source valid code now, hence [CSharpReplaceDefaultLiteralCodeFixProvider](http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp.Features/ReplaceDefaultLiteral/CSharpReplaceDefaultLiteralCodeFixProvider.cs,5858f2abee872edf) is not triggered?

Reply:
No, `case default:` is still illegal. The behavior change here is a casualty of binding change: the compiler no longer tries to give `default` a type via conversion, so the IDE doesn't get a type to to drive the refactoring.

I took another look and managed to tweak that, which allowed to revert most of the IDE test changes. Thanks!